src/test/components/__snapshots__/MenuButtons.test.js.snap

```diff
@@ -2050,7 +2050,7 @@ exports[`app/MenuButtons <Publish> matches the snapshot for the opened panel for
           class="menuButtonsDownloadSize"
         >
           (
-          1.55 kB
+          1.56 kB
```

Comment:
Hmm interesting, I don't know why we needed to update these values, but probably just some weird floating point issues?